metrics/src/metrics.rs

```diff
@@ -88,25 +88,31 @@ impl Default for MetricsAgent {
         Self::new(
             Arc::new(InfluxDbMetricsWriter::new()),
             Duration::from_secs(10),
+            //max per-second datapoint submission limit
+            1000
```

Comment:
Do you know if we hit this limit already when running a testnet with bench-tps?

Reply:
Unknown yet.  Will get hard numbers shortly

Reply:
Just ran a local single node with default settings and saw an average datapoint write rate up to 3,500-4,00 per second

Reply:
~4000 datapoints a second without bench-tps running?

Reply:
With bench tps running via client.sh